lib/src/widgets/chat/ds_image_message_bubble.widget.dart

```diff
@@ -102,64 +104,112 @@ class DSImageMessageBubble extends StatelessWidget {
   @override
   Widget build(BuildContext context) {
     return DSMessageBubble(
+      defaultMaxSize: 360.0,
```

Comment:
There's a reason of why we're considering maximum constraints with 360px for images instead of 480px?

Reply:
yes, according to PD, 480px is too big a size for the image bubble.